CHANGELOG.md

```diff
@@ -36,6 +36,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - [#7120](https://github.com/apache/trafficcontrol/pull/7120) *Docs* Update t3c documentation regarding parent.config parent_retry.
 
 ### Fixed
+- [#7293](https://github.com/apache/trafficcontrol/issues/7293) *Traffic Ops* Fixed Internal Server Error on GETting `/sslkeys_expirations`.
```

Comment:
This bug came from unreleased changes, so it doesn't need a Changelog entry because nothing will have actually changed in this regard between releases